gpflow/base.py

```diff
@@ -117,14 +116,27 @@ def __init__(
         therefore we need a positive constraint and it is natural to use constrained values.
         A prior can be imposed either on the constrained version (default) or on the unconstrained version of the parameter.
         """
-        if transform is None:
-            transform = tfp.bijectors.Identity()
+        if isinstance(value, Parameter):
+            transform = transform if transform else value.transform
+            prior = prior if prior else value.prior
+            prior_on = prior_on if prior_on else value.prior_on
+            trainable = trainable if trainable is not None else value.trainable
+            name = name if name else value.bijector.name
```

Comment:
```suggestion
            transform = transform or value.transform
            prior = prior or value.prior
            prior_on = prior_on or value.prior_on
            name = name or value.bijector.name
            trainable = value.trainable if trainable is None else trainable
```
Perhaps just a matter of personal preference, but I tend to find the or operator a little more readable in these cases.